src/igvfd/schemas/changelogs/in_vitro_system.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`in_vitro_system.json`*
 
+### Minor changes since schema version 25
+
+* Extend `nucleic_acid_delivery` enum list to include `lipofectamine` and `electroporation`.
```

Comment:
I think each addition should be one line each according to the standards we set for changelogs.